### DISCUSSION TITLE
How to disable all the animations?

### DISCUSSION THREAD

**Discussion by ribtoks** [neovide/neovide#2326] [Q&A] [answered]
I just upgraded to 0.12.2 and now there're new animations (buffer splitting, opening) enabled by default.

[Configuration docs](https://neovide.dev/configuration.html) only mention the cursor trails etc. How to disable these new animations and maybe there's a way to disable all animations?

(Ubuntu 22.04, Gnome, AppImage-based, neovim v0.9.2)

**Comment by emonadeo** [comment, marked as answer]
Run with `--no-multigrid`, or put `no-multigrid = true` into your `~/.config/neovide/config.toml`.

**Reply by ribtoks** [reply]
Thank you!
